include/frozen/map.h

```diff
@@ -352,6 +361,58 @@ constexpr auto make_map(std::array<std::pair<T, U>, N> const &items, Compare con
   return map<T, U, N, Compare>{items, compare};
 }
 
+template <typename T, typename U, typename Compare, std::size_t... KNs,
+          std::enable_if_t<!bits::is_pair<Compare>::value>* = nullptr>
+constexpr auto make_map(
+  Compare const& compare,
```

Comment:
Note that this has the comparison functor as it's first parameter instead of the last because otherwise this overload is sometimes ambiguous with the overload without comparison functor. (Probably due to the parameter pack).